.github/workflows/rust.yml

```diff
@@ -252,11 +252,15 @@ jobs:
         with:
           toolchain: stable
           components: rustfmt
-      - run: cargo fmt --manifest-path ${{ matrix.path }} --all -- --check
       - run: cargo build --manifest-path ${{ matrix.path }}
       - run: cargo test --manifest-path ${{ matrix.path }}
       - if: ${{ contains(matrix.path, 'core/Cargo.toml') }}
         run: cargo test --manifest-path ${{ matrix.path }} --features mock
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
+          components: rustfmt
+      - run: cargo fmt --manifest-path ${{ matrix.path }} --all -- --check
```

Comment:
what are these bits about though

Reply:
Install nightly channel with rustfmt component. Then, run cargofmt with it.

Reply:
Only night's fmt respect `rustfmt.toml` config file that's why.